frontend/src/component/common/NewConstraintAccordion/ConstraintAccordionEdit/ConstraintAccordionEditBody/RestrictiveLegalValues/RestrictiveLegalValues.tsx

```diff
@@ -162,22 +164,18 @@ export const RestrictiveLegalValues = ({
                 <ConstraintFormHeader>
                     Select values from a predefined set
                 </ConstraintFormHeader>
-                <ConditionallyRender
-                    condition={newContextFieldsUI}
-                    show={
-                        <Button variant={'text'} onClick={onSelectAll}>
-                            {isAllSelected ? 'Unselect all' : 'Select all'}
-                        </Button>
-                    }
-                />
+                <Button variant={'text'} onClick={onSelectAll}>
+                    {isAllSelected ? 'Unselect all' : 'Select all'}
+                </Button>
             </StyledStack>
             <ConditionallyRender
                 condition={legalValues.length > 100}
                 show={
                     <>
                         <ConditionallyRender
                             condition={
-                                Boolean(newContextFieldsUI) && Boolean(values)
+                                !disableShowContextFieldSelectionValues &&
+                                Boolean(values)
```

Comment:
There's no else condition here, so can this nested conditional be made into a single one?

E.g.

```ts
condition={legalValues.length > 100 && !disableShowContextFieldSelectionValues && Boolean(values)}
```

Might wanna extract that to a separate variable, though? 💁🏼

Reply:
We want to show the search when above 100 values.  The second conditional renders the selected values above it.

Reply:
Ah, you're righ! Sorry, for some reason, I didn't see the `show` on line 182 😅  All good 😄